sdk/digitaltwins/Azure.DigitalTwins.Core/tests/DigitalTwinTests.cs

```diff
@@ -49,7 +48,14 @@ public async Task DigitalTwins_Lifecycle()
 
                 // update twin
                 string updateTwin = TestAssetsHelper.GetRoomTwinUpdatePayload();
-                await client.UpdateDigitalTwinAsync(roomTwinId, updateTwin).ConfigureAwait(false);
+
+                // create request options to force the update using "*" ifMatch value
+                var reqeustOptions = new RequestOptions()
+                {
+                    IfMatch = new ETag("*")
```

Comment:
It is good to have an example using *, but I'd feel better if our test coverage included an actual ETag value.